Scripts/demand/freight.py

```diff
@@ -15,17 +15,34 @@ def calc_freight_traffic(self, mode):
         production_forecast = self.generate_trips(zone_data_forecast, mode)
         # Remove zero values
         base_mtx = self.base_demand.get_data(mode).clip(0.000001, None) 
-        external_growth = 1.1
-        production_calib = base_mtx.sum(1)
-        production = self.calibrate(production_calib[:len(production_base)], 
+        production = self.calibrate(base_mtx.sum(1), #[:len(production_base)], 
                                     production_base, 
                                     production_forecast)
-        # External demand is generated for the last centroids
-        external_base = production_calib[len(production):]
-        external_production = external_growth * external_base
-        production = numpy.append(production, external_production)
-        # Add harbour traffic!
-        demand = self.fratar(production, base_mtx)
+        # # External demand is generated for the last centroids
+        # external_base = production_calib[len(production):]
```

Comment:
I would just remove commented code, we can always check it out of version control if needed.